src/app/components/MostRead/README.md

```diff
@@ -0,0 +1,17 @@
+# MostRead
```

Comment:
Are we planning to fill out the README at this stage?

Reply:
We have another ticket relating to component health, so we were planning to tackle it as part of that.